### PR TITLE
XRT-2585: Add Canbus examples

### PR DESCRIPTION
**Pull request by Tylerg2471**

### CODE REVIEW

File: DeviceServices/canbus/deployment/config/pool.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "Logger":"logger",
+  "Threads": 8
```
Comment (BinduRao2018) [inline DeviceServices/canbus/deployment/config/pool.json]: Any reason to set the threads to 8?

File: DeviceServices/canbus/commands/add_string_device.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+mosquitto_pub -t xrt/devices/canbus/request -m \
+'{
+  "client": "example",
+  "request_id": "1010",
+  "op": "device:add",
+  "type": "xrt.request:1.0",
+  "device": "canbus-string-example",
+  "device_info": {
+    "profileName": "canbus-profile-strings",
```
Comment (BinduRao2018) [inline DeviceServices/canbus/commands/add_string_device.sh:11]: trying to add a device without an existing profile - canbus-profile-strings. Is this ok?

**Review by BinduRao2018 — COMMENTED** [review]
@Tylerg2471, added in-line comments

File: DeviceServices/canbus/deployment/config/pool.json (hunk 1 above)
Comment (Tylerg2471) [reply DeviceServices/canbus/deployment/config/pool.json]: Updated to be the default by removing the Threads parameter

File: DeviceServices/canbus/commands/add_string_device.sh (hunk 2 above)
Comment (Tylerg2471) [reply DeviceServices/canbus/commands/add_string_device.sh:11]: added a note into the walkthrough section that to use the commands that include string update the profiles.json.

Comment (Tylerg2471) [reply DeviceServices/canbus/commands/add_string_device.sh:11]: The reason for having the two profiles is resources with string as the type aren't supported in the continuous read, so a seperate profile has examples for using strings and the same with the commands.

Comment (BinduRao2018) [reply DeviceServices/canbus/commands/add_string_device.sh:11]: @Tylerg2471, canbus-profile-strings.json isn't checked in.

**Comment by BinduRao2018** [comment]
@Tylerg2471, one outstanding comment -  canbus-profile-strings.json isn't checked in.
Also, it makes sense to add the hardware configuration that was used to test the DS.

**Review by BinduRao2018 — CHANGES_REQUESTED** [review]

**Comment by Tylerg2471** [comment]
> @Tylerg2471, one outstanding comment - canbus-profile-strings.json isn't checked in. Also, it makes sense to add the hardware configuration that was used to test the DS.

Fixed now uploaded the profile

**Review by BinduRao2018 — APPROVED** [review]
LGTM

**Pull request merged**